src/main/java/Todo.java

```diff
@@ -1,3 +1,7 @@
+/**
+ * Represent an item to do.
+ * Inherits from the Task class.
+ */
```

Comment:
All comment descriptions and overall structure are quite clear and I love the chatbot's name "boba". Strictly following the coding standard (correct format of method names, variable names, etc) and refactoring the code well improves the overall code to be even better.